meson.build

```diff
@@ -0,0 +1,190 @@
+project(
+  'cloud-init',
+  meson_version: '>=0.63.0', # rockylinux/9
+  license: 'GPL-3 OR Apache-2.0',
+  default_options: [
+    # The default can yield broken results.
+    'python.install_env=auto',
+  ],
+)
+system = host_machine.system()
+sysconfdir = get_option('sysconfdir')
+init_system = get_option('init_system')
+
+lib_exec_dir = join_paths(get_option('prefix'), get_option('libexecdir'), 'cloud-init')
+render_tmpl = './tools/render-template'
+
+pymod = import('python')
+python = pymod.find_installation('python3')
+
+# https://mesonbuild.com/FAQ.html#but-i-really-want-to-use-wildcards for downsteam drop-in files
+find = find_program('find')
+
+install_subdir(
+  'cloudinit',
+  install_dir: python.get_install_dir(),
+  install_tag: 'python',
+)
```

Comment:
I think we want an install_mode on this too - I think nothing should require execute or write permissions.

Reply:
agreed. added.